docs/GettingStartedOnLinux.md

```diff
@@ -0,0 +1,211 @@
+
+#### Installing React Native
+
+OK, with all of that out of the way, it's finally time to install React Native.
+
+For most users of this guide, you'll need to use `sudo` to install React Native as root:
+
+* `sudo npm install -g react-native-cli`
+
+For users who installed NPM through other means, you'll likely want to install React Native as a normal user:
+
+* `npm install -g react-native-cli`
+
+## Create Project
+
+Hooray! React Native is installed.
+
+Now it's time to do what we came here to do: make an app!
+
+To create an app project, you'll need to sit down and input this command:
+
+* `react-native init AwesomeProject`
+
+It'll take several minutes, so don't panic if your terminal sits around for a while before spitting out any output.
+
+## Setting Up an Android Device
+
+OK, now that we have a starter project waiting for us, we're almost ready to run the code.
+
+But before we do, we'll need either an emulator or a real Android device to run our code on.
+
+In this tutorial, we're going to use a real Android device.
+
+First thing is to plug in your device and check the manufacturer code by using `lsusb`:
+
+```bash
+$ lsusb
+Bus 002 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
+Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+Bus 001 Device 003: ID 22b8:2e76 Motorola PCS 
+Bus 001 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
+Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
+Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+```
+
+Seems like a bunch of gibberish, eh? You want the line that represents your phone. If you're in doubt, try unplugging your phone and running the command again:
+
+```bash
+$ lsusb
+Bus 002 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
+Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+Bus 001 Device 002: ID 8087:0024 Intel Corp. Integrated Rate Matching Hub
+Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
+Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
+```
+
+You'll see that after removing the phone, the "Motorola PCS" disappeared from the list. This is the one that we care about!
+
+`Bus 001 Device 003: ID 22b8:2e76 Motorola PCS`
+
+From thabove line, you want to grab the first four digits from the device ID:
+
+`22b8:2e76` 
+
+In this case, it's `22b8`. That's the identifier for Motorola.
+
+You'll need to input this into your udev rules in order to get up and running.
+
+So, fire up your favorite text editor. For me it's vim:
+
+* `sudo vim /etc/udev/rules.d/51-android.rules`
+
+And add the following line to the file:
+
+```
+SUBSYSTEM=="usb", ATTR{idVendor}=="22b8", MODE="0666", GROUP="plugdev"
+```
+
+Just check that your device is properly connecting to ADB, the Android Debug Bridge, by using `adb devices`.
```

Comment:
Now check if your device is properly connected via ADB, by running `adb devices` in a Terminal.